MH4U Dex/HomeViewController.m

```diff
@@ -28,6 +30,21 @@ - (IBAction)resetButtonPressed:(id)sender
     [CoreDataController setShouldTriggerReloadUponRestart:YES];
 }
 
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    [MBProgressHUD showHUDAddedTo:self.view animated:YES];
+    dispatch_queue_t queue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
+    dispatch_async(queue, ^{
+        [[CoreDataController sharedCDController] tryLoadSequence];
+        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Usually, it's (very slightly) preferable to use
```objc
[[NSOperationQueue mainQueue] addOperationWithBlock:^{
    ...
}];
```
(because of a slight general preference for higher-level/Objective-C mechanisms over lower-level/pure-C mechanisms when both are available); there isn't a direct analog for the background-dispatch call, though.